packages/aws-lambda/src/r5/handler.ts

```diff
@@ -111,7 +111,9 @@ export function fhirSubscriptionHandler(
       return {
         statusCode: 404,
         body: JSON.stringify({
-          message: `Unable to find a subscription for ${subscription}`,
+          message: `Unable to find a subscription for ${JSON.stringify(
```

Comment:
Ouch - are you sure we are not leaking the Authorization header by doing this?

Reply:
that code was already there was it intentional to log [Object] here?